x-pack/plugins/lists/server/services/exception_lists/create_exception_list_item.ts

```diff
@@ -43,7 +43,7 @@ interface CreateExceptionListItemOptions {
   user: string;
   tags: Tags;
   tieBreaker?: string;
-  type: ExceptionListType;
+  type: ExceptionListItemType;
```

Comment:
Before both the list and item types were just `t.string` so it didn't scream at us, but now that the list type and item type differ, had to update.